asciicast2gif

```diff
@@ -4,7 +4,14 @@ set -e
 
 export DEBUG=${DEBUG:-0}
 
-asciicast2gif_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+path_to_self="${BASH_SOURCE[0]}"
+
+# Detect path
+if [ -L "$path_to_self" ]; then
```

Comment:
This checks whether parent dir is a symlink, correct?

Reply:
Yes it just checks if the *file* is a symlink. If it is than find the path to the original file.

Reply:
Actually please don't merge yet. It dosn't work. I was just lucky with the right folder depth when I tested with yarn

```bash
# npm:
ls -l /usr/local/bin/asciicast2gif
lrwxr-xr-x  1 seb  admin  47 26 Oct 17:52 /usr/local/bin/asciicast2gif -> ../lib/node_modules/asciicast2gif/asciicast2gif
# yarn:
ls -l /usr/local/bin/asciicast2gif
lrwxr-xr-x  1 seb  admin  79 26 Oct 17:55 /usr/local/bin/asciicast2gif -> ../../../Users/seb/.config/yarn/global/node_modules/asciicast2gif/asciicast2gif
```

Reply:
I think one last missing piece is adding these to `.npmignore`:

```
!/page/page.js
!/main.js
```

Reply:
I thought I had written a comment as to why this is not needed. Anyway once a .npmignore file is present the .gitignore file is not parsed. I should have realised this sooner, sorry for wasting your time. If you wish I can still add them.

>  If you want to include something that is excluded by your .gitignore file, you can create an empty .npmignore file to override it

[source](https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package)

Reply:
Ah, right, sorry! So that means we can remove page.js and main.js from repo, .gitignore them like it was before, but due to presence of .npmignore (where we don't ignore them) they would end up in the package.

Reply:
No need to apologise :) What I meant was that I thought I had written a comment after I pushed a commit (must have forgotten to actually press the green button). 
Yes exactly! I'll remove the main.js and page.js files in my next commit. 

FYI If you want to test what is send to npm you can run `npm pack`

Reply:
Awesome. Thanks for the tip, this will be my first npm published package so that's handy :)

Reply:
Cool! If there is nothing else, I think it is ready to be merged 🎉 .